src/fundus/scraping/crawler.py

```diff
@@ -146,6 +148,7 @@ def _build_article_iterator(
     def crawl(
         self,
         max_articles: Optional[int] = None,
+        max_seconds: Optional[float] = None,
```

Comment:
In general, I think `Int` as a type is a better fit here.

Reply:
Thanks @MaxDall, I have switched it to now use `Int`